Client/Frontend/Browser/Search/InitialSearchEngines.swift

```diff
@@ -56,7 +56,8 @@ class InitialSearchEngines {
     static let ddgDefaultRegions = ["DE", "AU", "NZ", "IE"]
     static let qwantDefaultRegions = ["FR"]
     static let yandexDefaultRegions = ["AM", "AZ", "BY", "KG", "KZ", "MD", "RU", "TJ", "TM", "TZ"]
-    static let ecosiaDefaultRegions = ["US", "GB", "FR", "DE", "NL", "BE", "CH", "SE"]
+    static let ecosiaDefaultRegions = ["AT", "AU", "BE", "BY", "CA", "DK", "ES", "FI", "GR", "HU", "IT",
```

Comment:
- I don't think `BY` is supposed to be in here; maybe copy/paste error?
- others look good!